File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
    /*vector<int> ans;
    for(int i=0;i<nums.size();i++){
        for(int j= i+1; j<nums.size();j++){
            if (nums[i]+nums[j] == target){
                ans.push_back(i);
                ans.push_back(j);
                return ans;
            } 
        }
    }
    return ans;*/
    unordered_map<int, int> m;
    vector<int> ans;
    for(int i=0;i<nums.size();i++){
        int first = nums[i];
        int second=target-first;
        if(m.find(second) != m.end()){
            ans.push_back(i);
            ans.push_back(m[second]);
            break;
        }
        m[first] = i;
    }
    return ans;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        result = [0,0]
        for i in range(len(nums)):
            for j in range(len(nums)):
                if(i != j and nums[i]+nums[j]==target):
                    result[0] = i
                    result[1] = j
        return result
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        